README.md

```diff
@@ -126,6 +126,49 @@ Try the following examples to learn more about Dapr's Java SDK:
 
 Please, refer to our [Javadoc](https://dapr.github.io/java-sdk/) website.
 
+#### Reactor API
+
+The Java SDK for Dapr is built using [Project Reactor](https://projectreactor.io/). It provides an asynchronous API for Java. When consuming a result is consumed synchronously, as in the examples above, the `block()` method is used.
+
+The code below does not make any API call, it simply returns the [Mono](https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html) publisher object:
+
+```java
+daprClient.publishEvent("mytopic", "my message");
+```
+
+To make an API call and invoke the result element (`void` becomes an empty result) synchronously, use the code below instead:
+```java
+daprClient.publishEvent("mytopic", "my message").block();
+```
+
+#### Serialization
+
+This SDK provides a basic serialization for request/response objects but also for state objects. Applications should provide their own serialization for production scenarios.
+
+1. Implement the [DaprObjectSerializer](https://dapr.github.io/java-sdk/io/dapr/serializer/DaprObjectSerializer.html) interface. See [this class](sdk-actors/src/test/java/io/dapr/actors/runtime/JavaSerializer.java) as example.
+2. Use your serializer class in the following scenarios:
+    * When building a new instance of [DaprClient](https://dapr.github.io/java-sdk/io/dapr/client/DaprClient.html):
+    ```java
+    DaprClient client = (new DaprClientBuilder())
+        .withObjectSerializer(new MySerializer()) // for request/response objects.
+        .withStateSerializer(new MySerializer()) // for state objects.
+        .build();
+    ```
+    * When registering an Actor Type:
+    ```java
+    ActorRuntime.getInstance().registerActor(
+      DemoActorImpl.class,
+      new MySerializer(), // for request/response objects.
+      new MySerializer()); // for state objects.
+    ```
+    * When building a new instance of [ActorProxy](https://dapr.github.io/java-sdk/io/dapr/actors/client/ActorProxy.html) to invoke an Actor instance:
+    ```java
+    ActorProxy actor = (new ActorProxyBuilder("DemoActor"))
+        .withObjectSerializer(new MySerializer()) // for request/response objects.
```

Comment:
Indicate the one used here must match the one used on the actor side.

Reply:
Nice. done.